frontend/public/components/nav.jsx

```diff
@@ -289,16 +289,20 @@ const MonitoringNavSection_ = ({urls, closeMenu}) => {
 };
 const MonitoringNavSection = connectToURLs(MonitoringRoutes.Prometheus, MonitoringRoutes.AlertManager, MonitoringRoutes.Grafana)(MonitoringNavSection_);
 
-const logout = e => {
-  e && e.preventDefault();
-  authSvc.logout();
-};
-
 const UserNavSection = connectToFlags(FLAGS.AUTH_ENABLED, FLAGS.OPENSHIFT)(({flags, closeMenu}) => {
   if (!flags[FLAGS.AUTH_ENABLED] || flagPending(flags[FLAGS.OPENSHIFT])) {
     return null;
   }
 
+  const logout = e => {
```

Comment:
It's hard because `authSvc` can't connect to redux to know it needs to delete the token, and I thought `deleteOpenShiftToken` and `logout` should be separate methods.

Reply:
That makes sense.  Can't both import the function from a separate file?  I realize its only 7 lines.

Reply:
It's also using a variable from the closure... Admittedly that could be passed in as a param.

If we're really worried about this, it would be better to fix it on the server so that logout knows it needs to delete the token, but that's a harder change because the auth provider isn't known in server.go and auth_openshift.go doesn't know anything about the API server.

Reply:
Sure, just asking the questions now to avoid duplication if we can help it. 
The `e && e.preventDefault && e.preventDefault()` will likely work in both cases?

Reply:
A utility function for logging out shouldn't do anything with click events.

I want to keep delete token and logout separate because there are times we need to call logout without deleting the token (on 401 errors from the API server)

Reply:
Agree!  Could wrap it when needed, `logout = (e) => { e.preventDefault();  util.logout() }`.  

No worries for now, can always revisit if we find we continue to monkey with it.